### PR TITLE
updated members page with my personal information

### PR DESCRIPTION
**Pull request by kamara-moses**
I added my RDS ID in id folder and added img and info in members folder.

### CODE REVIEW

File: ids/mapping.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   "khanjan": "Khanjan04",
   "kratika": "Kratika0907",
   "manjot": "manjots1607",
+  "moses": "moses-kamara",
```
Comment (Rucha1499) [inline ids/mapping.json:18]: As per mentioned in README.md this is the expected format - `[member_rds_id]: "<github_id>"`. So, please replace `moses-kamara` with your github id. 

For more information refer : https://github.com/Real-Dev-Squad/website-static/blob/main/README.md#ids

**Review by Rucha1499 — CHANGES_REQUESTED** [review]
Nice work! @kamara-moses 🎉 

Requesting minor changes. Thank you!

**Comment by Rucha1499** [comment]
The profile picture has to be square and face should cover 60% of the photo. Kindly have a look at : https://github.com/Real-Dev-Squad/website-static/blob/main/README.md#rules-for-profile-picture and make the changes accordingly. Thank you!

File: ids/mapping.json (hunk 1 above)
Comment (kamara-moses) [reply ids/mapping.json:18]: Ok. Will get this done today.

**Review by kamara-moses — COMMENTED** [review]
hey I made the changes can you check them out and let me know

**Review by Rucha1499 — CHANGES_REQUESTED** [review]
Please upload a square and a clear picture. Rest is all good! 👍

**Comment by kamara-moses** [comment]
> Please upload a square and a clear picture. Rest is all good! 👍

I uploaded another picture. Let me know how it looks.

**Review by swarajpure — CHANGES_REQUESTED** [review]
Thank you for the PR! 🎉 

Can you please remove the `members/moses/.DS_Store` file? It's not needed

Your current photo is a rectangle. Please crop it in a **square keeping your face in the center**.

**Comment by swarajpure** [comment]
Refer #93

**Pull request closed without merging**